src/index.js

```diff
@@ -471,7 +472,7 @@ class CopyPlugin {
                 `writing "${webpackTo}" to compilation assets from "${absoluteFrom}"`
               );
 
-              const info = { copied: true };
+              const info = { copied: true, absoluteFrom, relativeFrom };
```

Comment:
No need `absoluteFrom` and `relativeFrom`, we need only `originalFilename`

Reply:
`absoluteFrom` do you stats are unstable on different platforms

Reply:
What would qualify as `originalFilename` ? I would argue that this is `absoluteFrom`... The asset object has the following fields:

```
        absoluteFrom: '/home/flo/Projects/Javascript/copy-webpack-plugin/test/fixtures/directory/nested/nestedfile.txt',
        relativeFrom: 'nestedfile.txt',
        webpackTo: 'nested/nestedfile.txt',
        data: <Buffer >,
        targetPath: 'nested/nestedfile.txt',
        force: undefined
```

Reply:
Original filename - relative path where file was readed

Reply:
Given
```
├── css
│   ├── customera
│   │   └── styles.css
│   └── customerb
│       └── styles.css
└── webpack.config.js
```

I would need `css/customera/styles.css` and `css/customerb/styles.css` as documented in the info object. Is that would you mean?

Reply:
Based on your `context`

Reply:
Default contest is process.cwd(), so use you have `css/customera/styles.css`